Library/Homebrew/keg.rb

```diff
@@ -253,8 +253,13 @@ def python_pth_files_installed?
     Dir["#{path}/lib/python2.7/site-packages/*.pth"].any?
   end
 
+  def app_list(args = {})
+    app_prefix = (args[:prefer_opt] && optlinked?) ? opt_record : path
```

Comment:
> Any opinion on Dir[] vs. Pathname.glob()? The latter seems to be more popular nowadays.

I would prefer `Pathname.glob` because we prefer to using `pathname` over `string` for files.